src/compiler/moduleNameResolver.ts

```diff
@@ -1433,3 +1444,9 @@ namespace ts {
         return value !== undefined ? { value } : undefined;
     }
 }
+/* @internal */
+namespace ts {
+    export function startsWithSlash(name: string) {

```

Comment:
Since this function is not used anywhere else, you could avoid exporting it

Reply:
Great tip. I tried that first, but I had some trouble which seemed to be caused by the `/* internal */` comment, it could not resolve `CharacterCodes`. I just tried to move it again, and now it works. My knowledge of TypeScript is very low, so I will adjust it.